tests/Microsoft.Identity.Test.iOS.UIAutomation/iOSTests.cs

```diff
@@ -53,36 +52,38 @@ public void InitializeBeforeTest()
         /// </summary>
         [Test]
         [Category("FastRun")]
-        public async Task RunAllTestsAsync()
+        public void RunAllTests()
         {
-            var tests = new List<Func<Task>>()
+            var tests = new List<Action>()
             {
-                AcquireTokenTestAsync,
-                //AcquireTokenSilentTest,
-                AcquireTokenADFSV3InteractiveFederatedTestAsync,
-                AcquireTokenADFSV3InteractiveNonFederatedTestAsync,
-                AcquireTokenADFSV4InteractiveFederatedTestAsync,
-                AcquireTokenADFSV4InteractiveNonFederatedTestAsync,
-                AcquireTokenADFSV2019InteractiveFederatedTestAsync,
-                AcquireTokenADFSV2019InteractiveNonFederatedTestAsync,
+                AcquireTokenTest,
```

Comment:
This is not resetting internal caches such as instance discovery cache and OIDC cache.

Reply:
This hasn't been resolved.

Reply:
done.